framework/src/iotbus/iotbus_gpio.c

```diff
@@ -155,6 +156,58 @@ int iotbus_gpio_set_direction(iotbus_gpio_context_h dev, iotbus_gpio_direction_e
 	return IOTBUS_ERROR_NONE;
 }
 
+/**
+ * @brief Registers signal for current process based on rising or falling edge on the gpio line
+ */
+#ifndef CONFIG_DISABLE_SIGNALS
+int iotbus_gpio_register_signal(iotbus_gpio_context_h dev, iotbus_gpio_edge_e edge)
+{
+        int ret = -1;
+	FAR struct gpio_notify_s notify;
+
+        if (!dev)
+                return IOTBUS_ERROR_INVALID_PARAMETER;
+
+        switch (edge) {
+        case IOTBUS_GPIO_EDGE_NONE:
+		notify.gn_rising = false;
+		notify.gn_falling = false;
+                break;
```

Comment:
spaces. There are many places in this file. Please kindly check.

Reply:
Fixed